frontend/src/views/housing/project/ProjectView.vue

```diff
@@ -0,0 +1,436 @@
+const { getPermits, getSubmission } = storeToRefs(submissionStore);
+
+const typeStore = useTypeStore();
+const { getPermitTypes } = storeToRefs(typeStore);
+
+const assignee: Ref<User | undefined> = ref(undefined);
+const loading: Ref<boolean> = ref(true);
+const permitsNeeded = computed(() => {
+  return permitFilter({
+    permitNeeded: PermitNeeded.YES,
+    permitStatus: PermitStatus.NEW,
+    permits: getPermits.value,
+    permitTypes: getPermitTypes.value
+  })
+    .sort(permitNameSortFcn)
+    .sort(permitBusinessSortFcn);
+});
+const permitsNotNeeded = computed(() => {
+  return permitFilter({
+    permitNeeded: PermitNeeded.NO,
+    permits: getPermits.value,
+    permitTypes: getPermitTypes.value
+  })
+    .sort(permitNameSortFcn)
+    .sort(permitBusinessSortFcn);
+});
+const permitsSubmitted: ComputedRef<Array<CombinedPermit>> = computed(() => {
+  let firstFilter = permitFilter({
+    permitNeeded: PermitNeeded.YES,
+    permitStatus: PermitStatus.APPLIED,
+    permits: getPermits.value,
+    permitTypes: getPermitTypes.value
+  });
+  let secondFilter = permitFilter({
+    permitNeeded: PermitNeeded.YES,
+    permitStatus: PermitStatus.COMPLETED,
+    permits: getPermits.value,
+    permitTypes: getPermitTypes.value
+  });
+  return firstFilter.concat(secondFilter).sort(permitNameSortFcn).sort(permitBusinessSortFcn);
+});
+
+// Actions
+function permitBusinessSortFcn(a: CombinedPermit, b: CombinedPermit) {
+  return a.businessDomain > b.businessDomain ? 1 : -1;
+}
+function permitNameSortFcn(a: CombinedPermit, b: CombinedPermit) {
+  return a.name > b.name ? 1 : -1;
+}
+function permitFilter(config: PermitFilterConfig) {
+  const { permitNeeded, permits, permitStatus, permitTypes } = config;
+  let returnArray: Array<any> = permits;
+
+  if (permitNeeded) {
+    returnArray = returnArray.map((p) => {
+      const pType = permitTypes.find((pt) => pt.permitTypeId === p?.permitTypeId && p.needed === permitNeeded);
+      if (pType) return { ...p, ...pType };
+    });
+  }
+
+  if (permitStatus) {
+    returnArray = returnArray.map((p) => {
+      const pType = permitTypes.find((pt) => pt.permitTypeId === p?.permitTypeId && p.status === permitStatus);
+      if (pType) return { ...p, ...pType };
+    });
+  }
+
+  return returnArray.filter((pt) => !!pt) as Array<CombinedPermit>;
+}
+
+function displayTrackerStatus(authStatus: string | undefined, permitState: string | undefined): string {
+  if (permitState === PermitStatus.APPLIED) {
+    switch (authStatus) {
+      case PermitAuthorizationStatus.IN_REVIEW:
+      case PermitAuthorizationStatus.PENDING:
+        return authStatus;
+      case PermitAuthorizationStatus.NONE:
+      default:
+        return 'Submitted';
+    }
+  }
+  if (permitState === PermitStatus.COMPLETED) {
+    switch (authStatus) {
+      case PermitAuthorizationStatus.ISSUED:
+      case PermitAuthorizationStatus.DENIED:
+        return authStatus;
+      default:
+        return 'Completed';
```

Comment:
Could use `PermitStatus.COMPLETED` here to avoid a magic string. Not much to be done about the two Submitted strings unless we want to use the `IntakeStatus` enum there?

Reply:
I'm not certain we should. While `IntakeStatus` does have a 'Submitted' state and string, it's technically not about `IntakeStatus` On the very rare, off chance it needs to change it would affect here.